en_us.lang

```diff
@@ -157,200 +157,200 @@ gui.xaero_transfer_all=Transfer All Waypoints
-gui.xaero_box_entity_info=An interface displaying information about the living entity that you are pointing at including players.
+gui.xaero_box_entity_info=An interface displaying information about the living entity that you are pointing at, including players.
 gui.xaero_entity_info_settings=Entity Info Settings
-gui.xaero_entity_info_stay=Keep Entity Info For
+gui.xaero_entity_info_stay=Continue Displaying For
 gui.xaero_entity_info_distance=Reach Distance
-gui.xaero_entity_info_max_hearts=Max Amount of Hearts
-gui.xaero_current_biome=Show Current Biome
+gui.xaero_entity_info_max_hearts=Max Heart Number
+gui.xaero_current_biome=Display Current Biome
 gui.xaero_auto_map_size=Auto
-gui.xaero_reset_message=Are you sure you would like to reset default settings?
-gui.xaero_reset_message2=This action can't be reversed.
-gui.xaero_entity_info_numbers=Health In Numbers
+gui.xaero_reset_message=Are you sure you would like to restore the default settings?
+gui.xaero_reset_message2=This action cannot be undone!
+gui.xaero_entity_info_numbers=Health in Numbers
 gui.xaero_yaw=yaw
 gui.xaero_true=true
 gui.xaero_false=false
-gui.xaero_team_colours=Team Colours
+gui.xaero_team_colours=Team Colors
 gui.xaero_instant_waypoint=Quick Waypoint
 gui.xaero_entity_depth=Show Entity Depth
-gui.xaero_show_full_amount=Held Item Full Amount
-gui.xaero_show_flowers=Display Flowers
+gui.xaero_show_full_amount=Full Held Item Quantity
+gui.xaero_show_flowers=Show Flowers
 gui.xaero_show_entity_model=Display Entity Model
 gui.xaero_waypoint_names=WP Name Above Distance
-gui.xaero_waypoints_distance=Max WP Draw Dist.
+gui.xaero_waypoints_distance=Max WP Render Dist.
 gui.xaero_teleport_default_command=Default Teleport Command
-gui.xaero_waypoints_distance_min=Min WP Draw Dist.
+gui.xaero_waypoints_distance_min=Min WP Render Dist.
 gui.xaero_other_teams=Other Teams
-gui.xaero_entity_info_armour_numbers=Armour In Numbers
-gui.xaero_entity_info_armour=Armour Value
+gui.xaero_entity_info_armour_numbers=Armor in Numbers
+gui.xaero_entity_info_armour=Display Armor Info
 gui.xaero_arrow_scale=Arrow Scale
-gui.xaero_arrow_colour=Arrow Colour
-gui.xaero_smooth_dots=Smooth Dots
-gui.xaero_cave_zoom=Caves Zoom-in
-gui.xaero_lock_player_heads=Lock Player Heads
-gui.xaero_safe_mode_box=Use Java (CPU) based equivalent of the mod instead of OpenGL (GPU). Safe mode is a plan B, incase the mod fails normally. Not all features work in safe mode.
+gui.xaero_arrow_colour=Arrow Color
+gui.xaero_smooth_dots=Smooth Dot Edges
+gui.xaero_cave_zoom=Cave Mode Zoom In
+gui.xaero_lock_player_heads=Always Display Player Heads
+gui.xaero_safe_mode_box=Use Java-based (CPU) equivalent of this mod instead of the default, OpenGL-based (GPU) one. Safe mode is a plan B in case the mod does not function correctly. Not all features work in the safe mode. Do not use unless you have to.
 gui.xaero_height_limit=Entity Height Limit
 gui.xaero_world_map=World Map
 gui.xaero_use_world_map=Use World Map Chunks
```

Comment:
Is "World Map" here a proper noun? If so, what do you think about "Use World Map Mod Chunks"? Quite long, isn't it? :/

Reply:
At first glance it may seem like a good workaround, but I suppose that after such rename there may be people who will think that this option completely disables chunks on the fullscreen map. There were people asking for help because they accidentally pressed the H key, so you never know...

Reply:
But in that case they wouldn't want to disable the option, and that's a good thing. There is almost no reason to disable it.

Reply:
Hmm, that's sly. I like it, and I'm going to use it.